packages/components/src/keyboard-shortcuts/index.js

```diff
@@ -28,16 +23,18 @@ function KeyboardShortcut( {
 function KeyboardShortcuts( { children, shortcuts, bindGlobal, eventName } ) {
 	const target = useRef();
 
-	const element = map( shortcuts, ( callback, shortcut ) => (
-		<KeyboardShortcut
-			key={ shortcut }
-			shortcut={ shortcut }
-			callback={ callback }
-			bindGlobal={ bindGlobal }
-			eventName={ eventName }
-			target={ target }
-		/>
-	) );
+	const element = Object.entries( shortcuts ?? {} ).map(
```

Comment:
This is externally exposed API, so it could be nullish, therefore we're falling back to an empty object just in case.